input/InitialPopulations/compile/00_master.do

```diff
@@ -33,13 +33,15 @@ set matsize 1000
 
 /**************************************************************************************
 * DEFINE DIRECTORIES
-**************************************************************************************/
+*************************************************************************************/
 
 * Working directory
-global dir_work "C:\MyFiles\99 DEV ENV\JAS-MINE\data work\initial_populations"
+*global dir_work "C:\MyFiles\99 DEV ENV\JAS-MINE\data work\initial_populations"
+global dir_work "C:\Users\Patryk\Documents\SP_prep_pop"
```

Comment:
tiny side comment - at some point would it be helpful to make these more generic somehow? Either leaving as active directory where Stata is launched or prompting user to enter directory relative to do files where the process can find the data? I've tried to run it mostly as relative from a working directory when I start Stata to keep my own file paths out of the files (and they wouldn't work for anyone else!)